xep-0352.xml

```diff
@@ -24,6 +24,12 @@
   <shortname>csi</shortname>
   &mwild;
   <revision>
+	<version>0.2.1</version>
```

Comment:
You are using tabs here and everywhere in this file, while the common indent style used by XEPs is two spaces.